packages/timeline/package.json

```diff
@@ -35,6 +35,15 @@
     "semver": "^6.1.1"
   },
   "devDependencies": {
-    "flow-bin": "^0.101.0"
+    "@babel/core": "^7.5.4",
```

Comment:
Can you remove these dependencies on Babel please? They are already defined in the root `package.json`.

Reply:
If you do that then running rollup in the module will fail.  If someone wants to build just the module as an esm they can't as far as I can tell.  does having these deps here prevent the larger build from building correctly?

Reply:
Even if you just want to build one module, you need to install the dependencies from the root, as indicated in the setup: https://github.com/MainframeHQ/erebos#setup
All the packages share these dependencies and the same configuration (the `.babelrc` config points to the root one).